src/minesweeper/Config.py

```diff
@@ -0,0 +1,10 @@
+
+import pygame
+from ctypes.wintypes import RGB
+
+class Config:
```

Comment:
Rename the class to `gameConfig` to distinguish it from other potential configurations there might be.